examples/sinatra/README.md

```diff
@@ -0,0 +1,110 @@
+# Echo function example
+
+This example incudes an HTTP function that uses Sinatra to handle the request.
+It demonstrates how to use a web framework to assist you with writing HTTP
+functions.
+
+Note that while the Functions Framework and Google Cloud Functions _can_ be
```

Comment:
~Note that~ While the ..., it is not designed to...

Reply:
Done